stdlib/public/core/SIMDVectorTypes.swift.gyb

```diff
@@ -67,6 +85,7 @@ public struct SIMD${n}<Scalar> : SIMD where Scalar: SIMDScalar {
   }
 
 %  for (half,indx) in [('low','i'), ('high',str(n/2)+'+i'), ('even','2*i'), ('odd','2*i+1')]:
+  /// A half-length vector covering the ${half} elements of the vector.
```

Comment:
@stephentyrone Does this phrasing ("covering the even elements of the vector") sound okay to you? Suggestions?

Reply:
"made up of"?

Reply:
```suggestion
  /// A half-length vector made up of the ${half} elements of the vector.
```